resolwe_bio/processes/alignment/bowtie.yml

```diff
@@ -586,10 +587,16 @@
         {%  if trimming.trim_iter >= 1 %}
           samtools merge -h "${FW_NAME}_align_unsorted.sam" "${FW_NAME}_align_merged_unsorted.bam" ${FW_NAME}_*.bam
           re-checkrc
-          samtools sort -o "${FW_NAME}_align.bam" "${FW_NAME}_align_merged_unsorted.bam"
+          samtools sort \
+            -m {{ (requirements.resources.memory * 1024 * 1024 / requirements.resources.cores) | int }} \
```

Comment:
From samtools [docs](http://www.htslib.org/doc/samtools.html):
"-m INT - Approximately the maximum required memory **per thread**, specified either in bytes or with a K, M, or G suffix."

Reply:
I see. But in this case we should then set the number of threads as well because the same doc says
> -@ INT Set number of sorting and compression threads. By default, operation is single-threaded.

Reply:
Good point, fixed. Note that i also added `0.5` ratio to memory allocation. IMO, this is a conservative ratio, we can increase it at some later point.